requirements.txt

```diff
@@ -1,3 +1,3 @@
 beautifulsoup4==4.9.1
-git+https://github.com/phantomcyber/paramiko.git@main #paramiko with sha256
+git+https://github.com/phantomcyber/paramiko.git@soar-v1.0 #paramiko=2.9.2
```

Comment:
can we make the tag something like `2.9.2-fips`? the `soar-v1.0` tag doesn't really indicate what the version / change is in itself and could be misinterpreted to be tied to some sort of v1.0 of SOAR